.stylish-haskell.yaml

```diff
@@ -65,7 +63,7 @@ steps:
 
 # A common setting is the number of columns (parts of) code will be wrapped
 # to. Different steps take this into account. Default: 80.
-columns: 80
+columns: 120
```

Comment:
not really, except with current style the import lists are almost always spanning multiple lines, which I find annoying. There's a bit of settings in newest `stylish-haskell` so if the line is too long, imports could be aligned as
```hs
import           Data.List (foldl')
import qualified VeryLongNamespace.Data.Foo.Bar
                 (foo, bar, quux, longMethodName, anotherLongMethodName, Foobar,
                 MonadEveryhing, zyxxy)
```

or at least I remember making some such configuration possible.

I'll revert this change and investigate more (subjectively) pleasant (yet diff friendly) formatting style.